content/en/aws/kms/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Key Management Service (KMS)"
+linkTitle: "Key Management Service (KMS)"
+categories: ["LocalStack Community"]
+description: >
+  Key Management Service (KMS)
+---
+
+Key Management Service (KMS) is mainly about
+- creation and management of cryptographic keys
+- use of these keys to encrypt / decrypt data or to sign messages and to verify signatures for signed ones
+
+AWS documentation on KMS and all its operations can be found [here](https://docs.aws.amazon.com/kms/latest/APIReference/Welcome.html).
```

Comment:
Avoid using `here` or `this page`, since we want the user to know where they are being directed to.